envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/config/EnvoyControlTestConfiguration.kt

```diff
@@ -63,7 +64,7 @@ abstract class EnvoyControlTestConfiguration : BaseEnvoyTest() {
         var envoys: Int = 1
 
         // We use envoy version from master. This is 1.14.0-dev.
-        const val defaultEnvoyImage = "envoyproxy/envoy-alpine-dev:6c2137468c25d167dbbe4719b0ecaf343bfb4233"
+        const val defaultEnvoyImage = "envoyproxy/envoy-alpine-dev:5b1723ff54b1a51e104c514ee6363234aaa44366"
```

Comment:
Is comment above matching version?

Reply:
It is still 1.14.0 - but a couple of commits later when the fix was introduced.